hammer_cli.gemspec

```diff
@@ -23,7 +23,7 @@ EOF
   s.require_paths = ["lib"]
   s.executables = ['hammer']
 
-  s.add_dependency 'clamp', '~> 1.0.0'
+  s.add_dependency 'clamp', '~> 1.0'
```

Comment:
Is there a nice way we can exclude `1.1.0` from the range? It introduced some refactoring causing failures and it was taken back again in `1.1.1`